### PR TITLE
MRG: FIX: Remove regularize from examples

### PR DESCRIPTION
**Pull request by Eric89GXL**
Takes care of the 0.9 part of #1989 so we can hopefully release soon.

This will require an update to the sample dataset :( But in the meantime people can manually add the file from here:

http://staff.washington.edu/larsoner/sample_audvis-shrunk-cov.fif

### CODE REVIEW

**Comment by Eric89GXL** [comment]
Ready for review/merge from my end

**Comment by agramfort** [comment]
did you check if it has any impact on the example outputs?

**Comment by Eric89GXL** [comment]
There are some minor changes but they all look reasonable to me. It would be good if you checked, too, though, since I'm not 100% sure if the changes are all good. I couldn't check the lcmv volume one because plotting no longer works with latest `nibabel` and latest `nilearn`.

File: examples/inverse/plot_gamma_map_inverse.py [hunk 1]
```diff
@@ -35,9 +35,8 @@
 forward = mne.read_forward_solution(fwd_fname, surf_ori=True,
                                     force_fixed=False)
 
-# Read noise covariance matrix and regularize it
+# Read noise covariance matrix
```
Comment (dengemann) [inline examples/inverse/plot_gamma_map_inverse.py]: add a comment here about the precomputed reg maybe

File: examples/inverse/plot_lcmv_beamformer.py [hunk 2]
```diff
@@ -52,8 +52,6 @@
 forward = mne.read_forward_solution(fname_fwd, surf_ori=True)
 
 noise_cov = mne.read_cov(fname_cov)
-noise_cov = mne.cov.regularize(noise_cov, evoked.info,
-                               mag=0.05, grad=0.05, eeg=0.1, proj=True)
```
Comment (dengemann) [inline examples/inverse/plot_lcmv_beamformer.py]: redundancy helps to boost education, add a comment here as well.

File: examples/inverse/plot_lcmv_beamformer_volume.py [hunk 3]
```diff
@@ -54,9 +54,6 @@
 forward = mne.read_forward_solution(fname_fwd)
 
 noise_cov = mne.read_cov(fname_cov)
-noise_cov = mne.cov.regularize(noise_cov, evoked.info,
-                               mag=0.05, grad=0.05, eeg=0.1, proj=True)
-
```
Comment (dengemann) [inline examples/inverse/plot_lcmv_beamformer_volume.py]: and here...

**Comment by dengemann** [comment]
It would be good to add a comment. So people still get reminded of the reg topic.

File: mne/cov.py [hunk 4]
```diff
@@ -1211,9 +1211,9 @@ def regularize(cov, info, mag=0.1, grad=0.1, eeg=0.1, exclude='bads',
     channel type separately. Special care is taken to keep the
     rank of the data constant.
 
-    Note. This function is kept for reasons of backwards-compatibility.
-    Please consider explicitly using the `method` parameter in
-    compute_covariance to directly combine estimation with regularization
+    **Note:** This function is kept for reasons of backward-compatibility.
+    Please consider explicitly using the ``method`` parameter in
+    `compute_covariance` to directly combine estimation with regularization
```
Comment (dengemann) [inline mne/cov.py]: maybe add a link to the FAQ section.

**Comment by dengemann** [comment]
I'll pull and build docs to see where we are.

**Comment by Eric89GXL** [comment]
Comments addressed. @agramfort feel free to merge once you double-check the examples I touched.

**Comment by agramfort** [comment]
I have no time to check. Can somebody help? especially the results on the
sparse solvers...

**Comment by Eric89GXL** [comment]
Is it good enough if I post before and after images here?

**Comment by Eric89GXL** [comment]
(so that you can look and decide)

**Comment by agramfort** [comment]
yes

**Comment by Eric89GXL** [comment]
`master` on top, PR below

plot_gamma_map_inverse.py:

screenshot from 2015-05-07 08 06 29

01_pgmi_2


plot_lcmv_beamformer.py:

02_plb

02_plb_2


can't run plot_lcmv_beamformer_volume.py

plot_make_inverse_operator.py:

03_pmio

03_pmio_2


plot_mixed_norm_inverse.py:

04_pmni

04_pmni_2


plot_tf_lcmv.py:

05_ptl

05_ptl_2


plot_time_frequency_mixed_norm_inverse.py:

06_ptfmni

06_ptfmni_2

**Comment by agramfort** [comment]
thx

so the Gamma-MAP looks very different and worse than in master.

The regularization should be maintained only for this example.

btw I am not surprised.

**Comment by Eric89GXL** [comment]
How is it worse? I don't really know what to expect as correct -- but my impression is that it finds the one primary source and a couple of other secondary ones in both cases...?

**Comment by agramfort** [comment]
it misses the secondary ventral sources that peak later

**Comment by Eric89GXL** [comment]
Thanks -- reverted that example. Okay if I merge once Travis is happy?

**Comment by agramfort** [comment]
fine with me. thanks @Eric89GXL !

**Comment by adykstra** [comment]
The mixed-norm example actually looks better imo ...

**Comment by agramfort** [comment]
> The mixed-norm example actually looks better imo ...
> 

;)

**Comment by dengemann** [comment]
The gamma map should incorporate our cov estimation into it's hyperparameter estimation ....

;)

**Pull request merged**